### ISSUE TITLE
ALMA-Fix: Clean-Up `related`

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1507]
I deleted "Erscheint auch als" and did not exclude $i completly since some format/medium info for note is sometimes placed there. I also open another ticket for cleaning up "related" since there is duplicate info e.g. with `primarySource` and `fulltextonline` and others.

_Originally posted by @TobiasNx in https://github.com/hbz/lobid-resources/issues/1501#issuecomment-1305713871_
      

=> related references multiple other resources that also could be subsumed under primaryForm and other relational elements. This cleanup should be done after #1400

**Comment by TobiasNx** [comment]
Comment by:

```
The note is actually a note on the relation of the two resources but as we put it now, it is a note on the linked resource itself. I only see one option to solve this without an API break: Leave out the note in the transformation.

However, for some relations, we do have a specific property (which can be viewd as kind of sub-properties of related).

    for links to reproductions we could use hasVersion (and reproductions can link back with primaryForm)
    for links to "Online-Ausgabe" we could use fulltextOnline
```

The links in Online-Ausgabe are not links as fulltextOnline but to the records within lobid or with some other kind of identifier.. It therefore makes sense to separate these relations.

**Comment by TobiasNx** [comment]
Closing since Issue with `primaryForm`is solved. Other problems are not very specific.
